manifests/repo.pp

```diff
@@ -21,7 +21,7 @@
     case $facts['os']['family'] {
       'RedHat': {
         $majorrelease = $facts['os']['release']['major']
-        if (versioncmp(fact('os.release.major'), '7') >= 0 and $zabbix_version == '7.0') {
+        if (versioncmp(fact('os.release.major'), '8') >= 0 and $zabbix_version == '7.0') {
```

Comment:
if we don't support EL7, do we need the check for `versioncmp(fact('os.release.major'), '8'`?